onnx/defs/object_detection/defs.cc

```diff
@@ -18,19 +18,30 @@ quantizations while converting from original image into feature
 map and from feature map into RoI feature; in each ROI bin,
 the value of the sampled locations are computed directly
 through bilinear interpolation.
+
+The operator [crop_and_resize](https://www.tensorflow.org/api_docs/python/tf/image/crop_and_resize)
+in tensorflow behaves similarly as RoiAlign, where
+the major difference is the coordinate location and the mode chosen to compute the output tensor.
+This RoiAlign supports both crop_and_resize and RoiAlign.
 )DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     RoiAlign,
-    10,
+    11,
     OpSchema()
       .SetDoc(RoiAlign_ver1_doc)
+      .Attr(
+          "method",
```

Comment:
Most other OP use `mode` instead of `method`

Reply:
> Most other OP use mode instead of method

We already have an attribute called `mode`, used to distinguish the mode in `roi_align`. If we change `method` to `mode`, then what should the old `mode` be called?